### PR TITLE
[KYUUBI #4658][FOLLOWUP] Improve unit tests

### PR DESCRIPTION
**Pull request by Karsonnel**
### _Why are the changes needed?_

To allow up the reviewer's comment in  https://github.com/apache/kyuubi/issues/4660.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by Karsonnel** [comment]
Dear sir, the follow up about pr https://github.com/apache/kyuubi/pull/4660 will be do in this pr, please help to review. @yaooqinn @pan3793 @bowenliang123

File: extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala [hunk 1]
```diff
@@ -721,7 +721,7 @@ class HiveCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
             s"""INSERT OVERWRITE DIRECTORY '/tmp/test_dir' ROW FORMAT DELIMITED FIELDS
                | TERMINATED BY ','
                | SELECT * FROM $db1.$table;""".stripMargin)))
-      assert(e1.getMessage.contains(s"does not have [select] privilege on [$db1/$table/id"))
+      assert(e.getMessage.contains(s"does not have [select] privilege on [$db1/$table/id"))
```
Comment (bowenliang123) [inline extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala:724]:
```suggestion
      assert(e.getMessage.contains(s"does not have [select] privilege on [$db1/$table/id]"))
```

Comment (Karsonnel) [reply extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala]: fixed, thank you

**Comment by bowenliang123** [comment]
Let's change the name of test from `[KYUUBI #4658] INSERT OVERWRITE DIRECTORY did check query permission` to `INSERT OVERWRITE DIRECTORY`.
And add tests for both `InsertIntoDataSourceDirCommand` and `InsertIntoHiveDirCommand` in separate ut?

**Comment by Karsonnel** [comment]
> Let's change the name of test from `[KYUUBI #4658] INSERT OVERWRITE DIRECTORY did check query permission` to `INSERT OVERWRITE DIRECTORY`. And add tests for both `InsertIntoDataSourceDirCommand` and `InsertIntoHiveDirCommand` in separate ut?

> Let's change the name of test from `[KYUUBI #4658] INSERT OVERWRITE DIRECTORY did check query permission` to `INSERT OVERWRITE DIRECTORY`. And add tests for both `InsertIntoDataSourceDirCommand` and `InsertIntoHiveDirCommand` in separate ut?

rename test is ok, and i has added tests for both commands in https://github.com/apache/kyuubi/pull/4660, it's that enough?

**Comment by bowenliang123** [comment]
> > Let's change the name of test from `[KYUUBI #4658] INSERT OVERWRITE DIRECTORY did check query permission` to `INSERT OVERWRITE DIRECTORY`. And add tests for both `InsertIntoDataSourceDirCommand` and `InsertIntoHiveDirCommand` in separate ut?
> 
> rename test is ok, and i has added tests for both commands in #4660, it's that enough?

Have notice 2 ut added in PrivilegesBuilderSuite for different command plan , but only one e2e test added in HiveCatalogRangerSparkExtensionSuite. So it seems only one plan tested in e2e test.

**Comment by Karsonnel** [comment]
> > > Let's change the name of test from `[KYUUBI #4658] INSERT OVERWRITE DIRECTORY did check query permission` to `INSERT OVERWRITE DIRECTORY`. And add tests for both `InsertIntoDataSourceDirCommand` and `InsertIntoHiveDirCommand` in separate ut?
> > 
> > 
> > rename test is ok, and i has added tests for both commands in #4660, it's that enough?
> 
> Have noticed 2 ut added in PrivilegesBuilderSuite for different command plan , but only one e2e test added in HiveCatalogRangerSparkExtensionSuite. So it seems only one plan tested in e2e test.

Got it, i will add a e2e test for InsertIntoDatasourceDirCommand. thank you

**Review by bowenliang123 — APPROVED** [review]
LGTM. Let's wait for the CI builds.

**Review by cxzl25 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merged to master/1.7

**Pull request closed without merging**